src/router/index.js

```diff
@@ -18,12 +20,15 @@ function getRoutes(config) {
       component: () => import("../views/SelectDataSource.vue")
     });
     routes.push({
-      path: "/search/external/(.*)",
+      path: "/search/external/:pathMatch(.*)*",
```

Comment:
The migration proposes a different solution:

> /foo/(.*) becomes /foo/:_(.*)

see https://router.vuejs.org/guide/migration/#Removal-of-unnamed-parameters

Could this avoid the additional array checks?